android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -598,6 +597,14 @@ private void doMenuAction(int menuItemId) {
                 }
                 break;
         }
+
+        if(menuItemId != R.id.nav_schedule
```

Comment:
if there are finite elements in a set {b, c, d, e, f, g}

( a!= b && a!=c && a!=d && a!=e) `===` (a==f || a==g)

So your conditional can be simplified